### PR TITLE
[sqlalchemy] Initial integration

### PR DESCRIPTION
**Pull request by AdamKorcz**
Adds initial integration of [sqlalchemy](https://github.com/sqlalchemy/sqlalchemy).

This project is widely used in production by web apps.
Users of SQLAlchemy include ([src](https://www.sqlalchemy.org/#features)):

1. Yelp!
2. reddit
3. DropBox
4. The OpenStack Project
5. Survey Monkey

_________________________________________________________________________

@zzzeek Are you interested in integrating SQLAlchemy into OSS-fuzz?

This PR adds the build files and a fuzzer.

To complete the integration, a maintainers email address is needed for bug reports.

### CODE REVIEW

**Comment by zzzeek** [comment]
hey there -

not sure if fuzzing can easily apply to SQLAlchemy, can this tool simply post github issues?   my email is mike_mp at zzzcomputing.com

File: projects/sqlalchemy/sqlalchemy_fuzzer.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import atheris
+
+import sqlalchemy
+from sqlalchemy import create_engine
+from sqlalchemy import Table, Column, Integer, String, MetaData
+from sqlalchemy.sql import text
+
+def TestOneInput(input_bytes):
+    try:
+        sql_string = input_bytes.decode("utf-8")
```
[the remark this answers is not in the document]
Comment (AdamKorcz) [reply projects/sqlalchemy/sqlalchemy_fuzzer.py:27]: > what is "input_bytes" here

"input_bytes" is the pseudo-random data generated by the fuzzer.

**Comment by AdamKorcz** [comment]
> not sure if fuzzing can easily apply to SQLAlchemy

I will be happy to look at more entrypoints. Since SQLAlchemy takes untrusted input, it is worth fuzzing it continuously.

> can this tool simply post github issues? 

No

**Comment by AdamKorcz** [comment]
@inferno-chromium @jonathanmetzman @oliverchang @asraa @DavidKorczynski SQLAlchemy is ready for review!

**Review by jonathanmetzman — APPROVED** [review]
LGTM

**Pull request merged**